public/app/plugins/panel/text/TextPanel.tsx

```diff
@@ -32,6 +35,16 @@ export class TextPanel extends PureComponent<Props, State> {
     }
   }, 150);
 
+  componentDidMount() {
+    this.eventSub = this.props.eventBus.$on(LegacyGraphHoverEvent, event => {
+      console.log('grap hover event', event);
+    });
+  }
+
+  componentWillUnmount() {
+    this.eventSub?.unsubscribe();
+  }
+
```

Comment:
Testing this from a react panel. Currently there is no automatic unsubscribe. For panels that is written as a functional component I think we can create a hook that automatically handled unsubscribe on unmount